lib/common/strings.c

```diff
@@ -80,6 +83,34 @@ scan_ll(const char *text, long long *result, char **end_text)
     return rc;
 }
 
+/*!
+ * \internal
+ * \brief Scan a long long integer value from a string
+ *
+ * \param[in]  text           The string to scan (may be NULL)
+ * \param[out] result         Where to store result (or NULL to ignore)
+ * \param[in]  default_value  Value to use if text is NULL or invalid
+ *
+ * \return Standard Pacemaker return code
+ */
+int
+pcmk__scan_ll(const char *text, long long *result, long long default_value)
+{
```

Comment:
This function would be a great one to add unit tests for.

Reply:
Agreed ... Grace will be learning the unit test framework before long, and I figure these will be good test cases. Hopefully before 2.1.0 final release :)